README.md

```diff
@@ -1,18 +1,55 @@
 # Irving Example Theme
+## Description
+The **Irving Example Theme**'s purpose is to demo Irving and WordPress working together, and offer concrete examples to developers looking to learn Irving's patterns and architecture.
+ 
+It primarily contains examples of how to structure components in templates and template parts.
 
-This theme serves as an example of how an Irving theme can/should be setup, and how some common UI/UX can be approached.
+## Demo
+[See the theme in action](https://irving-production.herokuapp.com/).
+*(server may need to boot up, refresh if you see an error)*
 
-# Branch Workflow
-Tbd.
+## Templates
+[Documentation on templating with WP Irving](http://storybook.irvingjs.com).
```

Comment:
I assume this link is just placeholder for now? Might be worth adding a short bit of text explaining the high-level concept of templates here.

Reply:
Added. Thanks.